.github/workflows/deploy-pages.yml

```diff
@@ -54,36 +64,70 @@ jobs:
         with:
           name: conformance-${{ steps.get-details.outputs.name }}.json
           path: ./output.json
-  aggregate:
-    runs-on: "ubuntu-latest"
+  # https://github.com/actions/starter-workflows/blob/4a8f18e34dd13d2b6ee4d8da2ba72629eafe1609/pages/hugo.yml#L1
+  build:
+    runs-on: ubuntu-latest
+    env:
+      HUGO_VERSION: 0.117.0
     needs: [pull-outputs]
-    # the tests might have failed
-    if: always()
-    defaults:
-      run:
-        shell: bash
     steps:
-      - uses: actions/checkout@v3
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: ${{ env.HUGO_VERSION }}
+          extended: true
+      - name: Checkout
+        uses: actions/checkout@v3
         with:
-          path: "gateway-conformance"
+          submodules: recursive
```

Comment:
Do we have any submodules to checkout?

```suggestion
```

Reply:
We don't right now, but it's quite common to use submodule with hugo themes